### PR TITLE
feat(tier4_debug_rviz_plugin): pie chart with float32 multi array stamped

### PR DESCRIPTION
**Pull request by takayuki5168**
## Description

pie chart from tier4_debug_msgs::msg::Float32MultiArrayStamped based on jsk_rviz_plugins (BSD license)

image

This plugin is useful to visualize the data of debug messages like obstacle_cruise_planner's, longitudinal_controller's, lateral_controller's, etc.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: common/tier4_debug_rviz_plugin/src/jsk_overlay_utils.cpp [hunk 1]
```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2014, JSK Lab
```
Comment (takayuki5168) [inline common/tier4_debug_rviz_plugin/src/jsk_overlay_utils.cpp:1]: This file is included in some packages in autoware.universe.
This should be refactored with another PR.

https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_planning_rviz_plugin/src/tools/jsk_overlay_utils.cpp
https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_vehicle_rviz_plugin/src/tools/jsk_overlay_utils.cpp

**Comment by takayuki5168** [comment]
@yukkysaito @tkimura4 Could you review this PR.

File: common/tier4_debug_rviz_plugin/package.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>tier4_debug_rviz_plugin</name>
+  <version>0.1.0</version>
+  <description>The tier4_debug_rviz_plugin package</description>
+  <maintainer email="takayuki.murooka@tier4.jp">Takayuki Murooka</maintainer>
+  <license>Apache License 2.0</license>
```
Comment (yukkysaito) [inline common/tier4_debug_rviz_plugin/package.xml:8]: Is it ok?
The original license is BSD.
https://github.com/jsk-ros-pkg/jsk_visualization/blob/master/jsk_visualization/package.xml#L14

Comment (yukkysaito) [reply common/tier4_debug_rviz_plugin/package.xml:8]: I'll ask someone.

Comment (takayuki5168) [reply common/tier4_debug_rviz_plugin/package.xml:8]: https://github.com/autowarefoundation/autoware.universe/pull/2055/commits/5ed38fd4497d05612ee2e9a7593b565f80c45d6a
DONE

**Review by yukkysaito — APPROVED** [review]
LGTM
